src/view/calendar/custom-event-wrapper.jsx

```diff
@@ -95,9 +97,9 @@ export const CustomEventWrapperStyler = styled.div`
 	}
 `;
 
-export default function CustomEventWrapper({ event, children, selected }) {
+export default function CustomEventWrapper({ event, children }) {
 	const anchorRef = useRef();
-	const [open, setOpen] = useState(selected);
+	const [open, setOpen] = useState(false);
```

Comment:
why is this false instead of selected?

Reply:
@nubsthead I checked the props by spreading it and found out select does not exist so I remove it

Reply:
Yes did ,it's not there